Pear/View/OnboardingSearchDropdownView.swift

```diff
@@ -31,26 +33,40 @@ class OnboardingSelectTableView: UITableView {
     }
 }
 
+enum OnboardingSearchDropdownType {
+    case places // search from Google Places API
+    case local // search locally
```

Comment:
```suggestion
    /// Search from Google Places API￼
    case places
    /// Search locally 
    case local
```

Why not make these doc comments?￼